test/suite/test_txn26.py

```diff
@@ -37,6 +37,9 @@ class test_txn26(wttest.WiredTigerTestCase):
     session_config = 'isolation=snapshot'
 
     def test_commit_larger_than_active_timestamp(self):
+        if not wiredtiger.diagnostic_build():
```

Comment:
I am assuming this is the only changes you have submitted after reopening the PR?

Reply:
Yes